### DISCUSSION TITLE
Firefly -> Bitpanda -> Funds not received

### DISCUSSION THREAD

**Discussion by DerSauso** [iotaledger/firefly#2661] [Help & Support]
Hi Community, 

yesterday evening I sent some IOTA from my firefly wallet to two different Bitpanda accounts.
The first batch didn´t went through till now. The second batch arrived at the Bitpanda wallet within minutes.

When I check the transactions in the firefly wallet I see that the first transaction has the internal adress (Wallet 1) as sending adress and another internal adress (Wallet 1)  as well as the external adress of the Bitpanda wallet as recipient (could that be right - 2 adresses as recipient? I only put the adress of the Bitpanda wallet into the recipient field. The wallet must have added the second internal adress automatically...)

The second transaction (which was correctly booked) just had the second internal adress (Wallet 1) as sending adress and just the external adress of the Bitpanda wallet as recipient.

From my (rookie) point of view the transaction seems to be ok as the tangle explorer says: "The message is referenced by a milestone, the transaction is included in the ledger." If necessary I could post the message ID (only one message ID which includes both transactions).

I already contacted the Bitpanda helpdesk but till now I didn´t receive an answer.

For your support I thank you very much in advance!

**Comment by DerSauso** [comment]
Problem solved! It was apparently an issue on Bitpanda side. 
Thank you anyway folks! :-)

**Reply by freeleux** [reply]
exactly the same trouble but Bitpanda told is not solving it for me...
